api/ticket/shared_drafts.rst

```diff
@@ -0,0 +1,1838 @@
+Shared drafts
+*************
+
+Show
+====
+
+Required permission: ``ticket.agent``.
+
+``GET``-Request sent: ``/api/v1/tickets/{ticket id}/shared_draft``
+
+Sample response (base64 coded inline image is cut off):
+
+.. code-block:: json
+   :force:
+
+   # HTTP-Code 200 OK
+
+   {
+      "shared_draft_id": 2,
+      "assets": {
+         "TicketSharedDraftZoom": {
+            "2": {
+               "id": 2,
+               "ticket_id": 58,
+               "new_article": {
+                  "body": "<div>Some text for a shared draft. <br></div><div><br></div><div>Some inline image:<img tabindex=\"0\" style=\"width: 1000px; max-width: 100%;\" src=\"data:image/jpeg;base64,/9j/4AAQSkZJRg[...]
```

Comment:
Cutting off withouth the final `",` confuses the syntax highlighting in the final output. Please correct this (`[...]",`)